<launch>

  <!-- Setup Baxter -->
  
  <!-- Re-enable+enable to robot -->
  <node pkg="baxter_uncertainty" type="enable_baxter.sh" name="enable_baxter" />

  <!-- Disable the sonars -->
  <node pkg="rostopic" type="rostopic" name="disable_sonars" args="pub /robot/sonar/head_sonar/set_sonars_enabled std_msgs/UInt16 0" />

</launch>
